### DISCUSSION TITLE
Is the OpenCRX alive?

### DISCUSSION THREAD

**Discussion by NonLinearThoughts** [opencrx/opencrx#103] [Q&A]
Hi, I was looking for an open source Java and I found this. I found the whole setup and the concept very nice. BUT I do not see it very alive. I have not found a single forum to deal with and get information. There is nothing related to communication on the site either, has many broken links and in many places it seems unmaintained .
Only here I found to communicate. It seems either the project is dead (although I saw a new version in September !!!) or there is no community. Pity

**Comment by opencrx** [comment]
Yes, it is. Please follow these [GitHub Community Guidelines](https://docs.github.com/en/github/site-policy/github-community-guidelines) if you expect answers.
